std/encoding.d

```diff
@@ -2233,7 +2101,7 @@ body
     }
     else static if(is(Src==AsciiChar))
     {
-        transcode!(char,Dst)(cast(string)s,r);
+        transcodeFromAscii(s, r);
```

Comment:
It has some narrow use. I'm not sure if it's justified here, probably not. My view on the situation can be [read here](https://jakobovrum.github.io/d/2016/01/20/memory-safety.html). If anyone has a better framework for `@trusted` please let me know.